lms/static/js/student_account/views/LoginView.js

```diff
@@ -231,12 +231,17 @@
             },
 
             renderAuthWarning: function() {
-                var message = _.sprintf(
-                    gettext('You have successfully signed into %(currentProvider)s, but your %(currentProvider)s' +
-                            ' account does not have a linked %(platformName)s account. To link your accounts,' +
-                            ' sign in now using your %(platformName)s password.'),
-                    {currentProvider: this.currentProvider, platformName: this.platformName}
-                );
+                var message = HtmlUtils.interpolateHtml(
+                        gettext('{paragraphStart}You have successfully signed into {currentProvider}, but your {currentProvider} account does not have a linked {platformName} account.{paragraphEnd}' + // eslint-disable-line max-len
```

Comment:
Can we do something like:
```suggestion
                        gettext('{paragraphStart}You have successfully signed into {currentProvider},' 
                        + ' but your {currentProvider} account does not have a linked {platformName} '
                        + 'account.{paragraphEnd}' + 
```

Reply:
@HamzaIbnFarooq Just copied the idea from the same file https://github.com/ucsd-ets/edx-platform/blob/ccb12146f7357abb73d24e0f475c40e9a787c77f/lms/static/js/student_account/views/LoginView.js#L155